hedera-node/data/config/bootstrap.properties

```diff
@@ -2,9 +2,12 @@ bootstrap.throttleDefsJson.resource=throttles-dev.json
 ledger.id=0x03
 staking.periodMins=1
 scheduling.whitelist=ConsensusSubmitMessage,CryptoTransfer,TokenMint,TokenBurn,CryptoApproveAllowance,CryptoUpdate
-hedera.workflows.enabled=
-hedera.states.enabled=false
+# All below properties should be set to appropriate values for testing using workflows
+# (Boolean values to true and workflows.enabled to the HAPI operations enabled with workflows)
 # Initial Service schemas require on-disk storage, so all these need to be false if enabling States
 accounts.storeOnDisk=false
 tokens.storeRelsOnDisk=false
 tokens.nfts.useVirtualMerkle=false
+# List of HAPI operations enabled to work with workflows.
+hedera.workflows.enabled=
+hedera.states.enabled=false
```

Comment:
Not a blocker for this PR, but I think we can remove the `hedera.states.enabled` property and use `MerkleHederaState` iff the `workflows.enabled` set is non-empty.

Reply:
Agreed.